lib/spack/docs/tutorial/examples/1.package.py

```diff
@@ -22,13 +22,12 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
-# flake8: noqa
```

Comment:
These files were intentionally left unchecked because they're examples for the tutorial.  I guess it's ok to remove `flake8: noqa` and make them conform, but if you look at `tutorial_sc16_packaging.rst`, you'll see that we key off of `flake8:noqa` so that the license doesn't clutter up the docs.  Can you fix that?  maybe add another string here, like `# Do not delete this line -- tutorial example starts below`?